source/includes/steps-general-connection-tab.yaml

```diff
@@ -0,0 +1,66 @@
+---
+title: Click on :guilabel:`Advanced Connection Options`.
+level: 4
+ref: compass-advanced-connection
+content: |
+  .. figure:: /images/compass/advanced-connection-options.png
+     :figwidth: 690px
+     :alt: New Advanced Connection Options
+---
+title: Select :guilabel:`General`.
+level: 4
+ref: select-general-tab
+content: |
+  .. procedure::
+  
+     .. step:: Select a Connection String Scheme.
+
+        You can select from the following two options:
```

Comment:
[minor]

Suggest: "You can select one of the following options: "

Reply:
Done.